v1/README_GCS_Avro_to_Cloud_Bigtable.md

```diff
@@ -21,11 +21,11 @@ on [Metadata Annotations](https://github.com/GoogleCloudPlatform/DataflowTemplat
 * **bigtableProjectId** : The ID of the Google Cloud project that contains the Bigtable instance that you want to write data to.
 * **bigtableInstanceId** : The ID of the Bigtable instance that contains the table.
 * **bigtableTableId** : The ID of the Bigtable table to import.
-* **inputFilePattern** : The Cloud Storage path pattern where data is located. (Example: gs://<BUCKET>/<FOLDER>/<PREFIX>*).
+* **inputFilePattern** : The Cloud Storage path pattern where data is located. (Example: gs://<BUCKET_NAME>/FOLDER/PREFIX*).
```

Comment:
Is the <> removing from folder and prefix intentional ?

Reply:
yes, it is removed intentionally. The text that is in <> is getting truncated in Readme page.